tests/genai/optimize/test_optimize.py

```diff
@@ -253,6 +253,39 @@ def optimize(self, eval_fn, dataset, target_prompts):
     assert len(result.optimized_prompts) == 1
 
 
+def test_optimize_prompts_warns_on_unused_prompt(
+    sample_translation_prompt, sample_summarization_prompt, sample_dataset, capsys
+):
+    mock_optimizer = MockPromptAdapter()
+
+    # Create predict_fn that only uses translation prompt, not summarization prompt
+    def predict_fn_single_prompt(input_text, language):
+        prompt = mlflow.genai.load_prompt("prompts:/test_translation_prompt/1")
+        prompt.template
```

Comment:
can we insert a comment here? This looks like we forgot to remove this line.

Reply:
Can we call format instead?